### PR TITLE
Blog: 2023 Review

### PR DESCRIPTION
**Pull request by psiinon**
<details><summary>Screenshot</summary>
<p>

Screenshot 2024-01-03 at 16-19-46 2023 in Review

</p>
</details>

### CODE REVIEW

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
@kingthorin or @ricekot ?

File: site/content/blog/2024-01-03-2023-in-review/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+date: "2024-01-03"
+authors: 
+    - simon
+---
+
+Happy New Year, and welcome to the ZAP review for 2023!
+
+### ZAP Chat Video
+
+This year we decided review the year as a [ZAP Chat](/zap-chat/) episode:
```
Comment (kingthorin) [inline site/content/blog/2024-01-03-2023-in-review/index.md:20]:
```suggestion
This year we decided to review the year as a [ZAP Chat](/zap-chat/) episode: 
```

**Review by kingthorin — COMMENTED** [review]
I guess we should also look at updating the roadmap page/content

**Comment by psiinon** [comment]
Change applied 👍

**Review by kingthorin — APPROVED** [review]

**Pull request merged**